notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<p>Should this heading be one level higher?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>